src/SPC/builder/unix/library/ldap.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPC\builder\unix\library;
+
+trait ldap
+{
+    protected function build(): void
+    {
+        $openssl_include = BUILD_INCLUDE_PATH;
+        $openssl_lib = BUILD_LIB_PATH;
+        shell()->cd($this->source_dir)
+            ->exec(
+                'CC="musl-gcc -I' . $openssl_include . ' -L' . $openssl_lib . '" LDFLAGS="-static" ./configure ' .
+                '--enable-static=yes ' .
+                '--disable-shared ' .
+                '--disable-slapd ' .
+                '--without-systemd ' .
+                '--with-tls=openssl ' .
+                '--prefix=' . BUILD_ROOT_PATH
+            )
```

Comment:
this doesn't work with the openssl build produced here, errors out with

`checking for ssl_ctx_set_ciphersuites in -lssl... no`

Doing it manually and supplying $openssl_install/include and $openssl_install/lib to the build ldap works.

```
#!/bin/bash

openssl_version=1.1.1w
openldap_version=2.6.6

# Build OpenSSL
wget https://openssl.org/source/openssl-$openssl_version.tar.gz
tar -xvf openssl-$openssl_version.tar.gz
cd openssl-$openssl_version
openssl_install=`pwd`/local

CC="musl-gcc -I$openssl_install/include/openssl -L$openssl_install/lib -static -idirafter /usr/include/ -idirafter /usr/include/x86_64-linux-gnu/" ./config no-shared -static --prefix=$openssl_install
make -j
make install
cd ..

# Build OpenLDAP
wget ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-$openldap_version.tgz
tar -xvf openldap-$openldap_version.tgz
cd openldap-$openldap_version

CC="musl-gcc -I$openssl_install/include -L$openssl_install/lib" LDFLAGS="-static" ./configure --disable-shared --disable-slapd --with-tls=openssl
make depend
make -j

cd ..

# Cleanup
rm openssl-$openssl_version.tar.gz
rm openldap-$openldap_version.tgz
```

Reply:
I think I might have to pass the psth to LDFLAGS too